packages/@o3r/localization/src/tools/localized-decimal.pipe.spec.ts

```diff
@@ -30,6 +30,25 @@ import {
   LocalizedDecimalPipe
 } from './localized-decimal.pipe';
 
+/**
+ * Fixture for ChangeDetectorRef
+ */
+class ChangeDetectorRefFixture implements Readonly<ChangeDetectorRef> {
```

Comment:
Fallback changes removed in previous PR because jest/unbound-method was not activated at this time